x-pack/legacy/plugins/siem/public/containers/case/use_delete_cases.test.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useDeleteCases, UseDeleteCase } from './use_delete_cases';
+import * as api from './api';
+
+jest.mock('./api');
+
+describe('useDeleteCases', () => {
+  const abortCtrl = new AbortController();
+  it('init', async () => {
+    await act(async () => {
+      const { result, waitForNextUpdate } = renderHook<string, UseDeleteCase>(() =>
+        useDeleteCases()
+      );
+      await waitForNextUpdate();
+      expect(result.current).toEqual({
+        isDisplayConfirmDeleteModal: false,
+        isLoading: false,
+        isError: false,
+        isDeleted: false,
+        dispatchResetIsDeleted: result.current.dispatchResetIsDeleted,
+        handleOnDeleteConfirm: result.current.handleOnDeleteConfirm,
+        handleToggleModal: result.current.handleToggleModal,
+      });
+    });
+  });
+
+  it('calls deleteCases with correct arguments', async () => {
+    const spyOnDeleteCases = jest.spyOn(api, 'deleteCases');
+
+    await act(async () => {
+      const { result, waitForNextUpdate } = renderHook<string, UseDeleteCase>(() =>
+        useDeleteCases()
+      );
+      await waitForNextUpdate();
+
+      result.current.handleOnDeleteConfirm([{ id: '1' }, { id: '2' }, { id: '3' }]);
```

Comment:
nit: As we have the mock caseIds several times in each case, I would recommend to put it as a variable, incase if the format is updated in the future